en/docs/learn/identifier-first-flow-handler.md

```diff
@@ -0,0 +1,91 @@
+
+![identifier-first](../assets/img/learn/identifier-first.png)
+
+## Let's try this out for a scenario!
+
+Suppose the admin wants the user who attempts to log in, to be authenticated by a federated authenticator that is chosen based on the domain name specified. Let's also assume that the user has opted to validate the identity before proceeding to authenticate the user. 
+
+Follow the steps given below to accomplish this. 
+
+!!! tip "Before you begin"
+    
+    1.  Install WSO2 IS 5.9.0 by downloading the [installer](https://wso2.com/identity-and-access-management/)
+
+    2.  Navigate to <IS_HOME>/repository/conf/deployment.toml file. Set the enable_email_domain configuration to true to enable email authentication. By doing this, we are configuring WSO2 IS to use the email as the username.
+
+        ```toml
+        [tenant_mgt]
+        enable_email_domain= true
+        ```
+
+    3.  [Download](https://tomcat.apache.org/download-80.cgi) and install Apache Tomcat version 8.*.* or above.
+
+    4.  Open the /etc/hosts file and add the following entry.
+
+        ```bash
+        127.0.0.1 localhost.com
+        ```
+    5.  Restart WSO2 Identity Server.
+
+1.  Download the saml2-web-app-pickup-dispatch.com.war file from [here](https://github.com/wso2/samples-is/releases) and copy it inside the <TOMCAT_HOME>/webapps directory.
+
+2.  Start the tomcat server.  
+
+3.  In the WSO2 IS [management console](https://localhost:9443/carbon/admin/login.jsp), create a new service provider and expand the Inbound Authentication configuration. Select the SAML2 Web SSO configuration section, and click Configure.
+
+4.  Provide the following details and register.
+    -   Issuer- \saml2-web-app-pickup-dispatch.com
+    -   Assertion Consumer URL- \http://localhost.com:8080/saml2-web-app-pickup-dispatch.com/consumer
+
+5.  Access the PickUp application URL at http://localhost.com:8080/saml2-web-app-pickup-dispatch.com. The login screen should only prompt for the username. Only after entering the correct username it proceeds to the next screen where you are requested to enter the password. 
+    ![only-username](../assets/img/learn/only-username.png)
+
+6.  Now let's try to incorporate a federated authenticator in this scenario. In the management console, configure the federated authenticators required. Details on how to configure these can be found [here](../../learn/configuring-federated-authentication/).  
```

Comment:
Now let's try to incorporate a federated authenticator **to** this scenario. For instructions on configuring a federated authenticator, see [Configuring Federated Authentication](../../learn/configuring-federated-authentication/).